src/I18n/Number.php

```diff
@@ -39,6 +39,20 @@ class Number
      */
     const FORMAT_CURRENCY = 'currency';
 
+    /**
+`     * Format type to format as currency, accounting style (negative numbers in parentheses)
+     *
+     * @var string
+     */
+    const FORMAT_CURRENCY_ACCOUNTING = 'currency_accounting';
+
+    /**
+     * ICU Constant for accounting format; not yet supported by INTL library
+     *
+     * @var int
+     */
+    const CURRENCY_ACCOUNTING = 12;
```

Comment:
This needs documentation on where the value comes from and why it would work without explicit support.